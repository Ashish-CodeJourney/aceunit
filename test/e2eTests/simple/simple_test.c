#include <assert.h>
#include <stdio.h>

#define printme() do { fprintf(stderr, "%s:%d: %s\n", __FILE__, __LINE__, __func__); } while (0)

void test1() {
    printme();
}

void test2() {
    printme();
}

void test3() {
    printme();
}
