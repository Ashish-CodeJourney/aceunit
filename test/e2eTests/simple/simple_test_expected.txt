simple_test.c:7: test1
simple_test.c:11: test2
simple_test.c:15: test3
AceUnit: 3 test cases, 3 successful, 0 failed.
